assets/src/edit-story/components/tablist/styles.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import {
+  Button,
+  BUTTON_VARIANTS,
+  Headline,
+  ThemeGlobals,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '../../../design-system';
+
+export const Tablist = styled.div`
+  width: 308px;
+  background: ${({ theme }) => theme.colors.bg.primary};
+`;
+Tablist.propTypes = {
```

Comment:
Smart! I think we should attach that to the styled component like you suggested here rather than expect people to add it every time it's used (even thought it's just gonna be once)

Reply:
i think this add got skipped?

Reply:
just added it 🤕